app/helpers/rubygems_helper.rb

```diff
@@ -161,6 +161,10 @@ def show_all_versions_link?(rubygem)
     rubygem.versions_count > 5 || rubygem.yanked_versions?
   end
 
+  def latest_release_date(rubygem)
+    (rubygem.latest_version || rubygem.versions&.last)&.built_at&.strftime("%B %d, %Y")
```

Comment:
Is there any case where `rubygem.latest_version` is `nil` (or `false`) but `rubygem.versions.last` is not? Also `created_at` should be used.

Reply:
I ran `Rubygem.left_outer_joins(:versions).where(versions: { id: nil })` and it returned 79 results. All of them had the indexed property set to `false` (Not sure if that affects the search result eligibility).

I followed the pattern from this method: 
https://github.com/rubygems/rubygems.org/blob/0c48ef3d8584e6cc3c8f28e9124f8e7c2be913b2/app/helpers/rubygems_helper.rb#L166

Reply:
Ended up removing the method and using `version_date_tag` instead